_docs/_latest/setup/upgrade.md

```diff
@@ -151,6 +158,271 @@ change a certain resource, you'll need to create a new deployment of Forseti.
+        ...
+    ```
+1. Rule files updates:  
+**No changes to the configuration file.**
+
+{% endcapture %}
+{% include site/zippy/item.html title="Upgrading 2.0.0 to 2.1.0" content=upgrading_2_0_0_to_2_1_0 uid=2 %}
+
+{% capture upgrading_2_1_0_to_2_2_0 %}
+
+1. Open cloud shell when you are in the Forseti project on GCP.
+1. Checkout forseti with tag v2.2.0 by running the following commands:
+    1. If you already have the forseti-security folder under your cloud shell directory, 
+    run command `rm -rf forseti-security` to delete the folder.
+    1. Run command `git clone https://github.com/GoogleCloudPlatform/forseti-security.git` to 
+    clone the forseti-security directory to cloud shell.
+    1. Run command `cd forseti-security` to navigate to the forseti-security directory.
+    1. Run command `git checkout tags/v2.2.0` to checkout version `tags/v2.2.0` of Forseti Security.
+1. Download the latest copy of your Forseti server deployment template file from the Forseti server GCS 
+bucket (located under `forseti-server-xxxxxx/deployment_templates`).
+1. Add the following fields to the compute engine section inside your deployment template.  
+    `region` - The region of your VM, e.g. us-central1,  
+    `vpc-host-project-id` - VPC host project ID, by default if you are not using VPC, 
+    you can default it to your Forseti project ID,  
+    `vpc-host-network` - VPC host network, by default if you are not using VPC, you can default it to `default`,  
+    `vpc-host-subnetwork`- VPC host subnetwork, by default if you are not using VPC, you can default it to `default`  
+    ```
+    
+    # Compute Engine
+    - name: forseti-instance-server
+      type: forseti-instance-server.py
+      properties:
+        # GCE instance properties
+        image-project: ubuntu-os-cloud
+        image-family: ubuntu-1804-lts
+        instance-type: n1-standard-2
+        
+
+        # ---- V2.2.0 newly added fields ----
+        region: **{YOUR FORSETI VM REGION, e.g. us-central1}**
+        vpc-host-project-id: {YOUR_FORSETI_PROJECT_ID}
+        vpc-host-network: default
+        vpc-host-subnetwork: default
+
+
+        ...
+        run-frequency: ...
+        
+    ```
+1. Update the `forseti-version` inside the deployment template to `tags/v2.2.0`.
+1. Create file `deploy-forseti-server-xxxxx-2-2-0.yaml` under `forseti-security/deployment-templates`.
+1. Copy and paste the content of the deployment template to file `deploy-forseti-server-xxxxx-2-2-0.yaml`.
+1. Upload file `deploy-forseti-server-xxxxx-2-2-0.yaml` back to the GCS bucket (`forseti-server-xxxxxx/deployment_templates`)
+1. Navigate to [Deployment Manager](https://console.cloud.google.com/dm/deployments) and 
+copy the deployment name for Forseti server.
+1. Run command `gcloud deployment-manager deployments update DEPLOYMENT_NAME --config deploy-forseti-server-xxxxx-2-2-0.yaml`  
+If you see errors while running the deployment manager update command, please refer to below section 
```

Comment:
This is not an `if`, as they will see this problem for sure.  So, how about moving the section from below, and put them here?

Reply:
The workaround we put in before in the upgrade instructions will  get user to v2.2.0 without updating the vm instance (staying on the old branch to do the update). They will still have the same problem updating to v2.3.0/v2.4.0 that's why I am trying to move the error instruction out so it can be used for other versions.